requirements.txt

```diff
@@ -2,3 +2,5 @@ tornado
 toolz
 zict
 six
+numpy
```

Comment:
These are optional deps I think so we might consider if they are needed or not.

Reply:
That is a good point. Let me remove them from requirements.txt and rather just install them with pip directly in the Dockerfile itself. 

If the conda PR merges at another point that could also be used instead to make the Dockerfile even cleaner.